amfTest/amfcfg.yaml

```diff
@@ -102,22 +102,13 @@ configuration:
 logger:
   AMF:
     debugLevel: info
-    ReportCaller: false
   NAS:
     debugLevel: info
-    ReportCaller: false
```

Comment:
We are removing reportCaller because new logging library does not need this right?

Reply:
Correct